test_runner/src/main/kotlin/ftl/reports/util/ReportManager.kt

```diff
@@ -176,7 +176,9 @@ object ReportManager {
 
         val oldTestResult = GcStorage.downloadJunitXml(args)
 
-        newTestResult.mergeTestTimes(oldTestResult)
+        if (args.useLegacyJUnitResult) {
+            newTestResult.mergeTestTimes(oldTestResult)
```

Comment:
I've decided to omit this call cause new junit result (from api) does not favor success over failure nor error.